build/_k-test.shade

```diff
@@ -3,6 +3,7 @@ use import="Environment"
 
 default NO_PARALLEL_TEST_PROJECTS='${E("NO_PARALLEL_TEST_PROJECTS")}'
 default KOREBUILD_TEST_DNXCORE='${E("KOREBUILD_TEST_DNXCORE")}'
+default CORECLR_RUNTIME_VERSION='${E("CORECLR_RUNTIME_VERSION")}'
```

Comment:
@moozzyk you're not unblocking DNX as this PR stands. Not sufficient to test CLR with the old DNX and Core with the new DNX.

Further if `cmd` has the problems you mentioned, failures when testing w/ the new DNX won't break the build.